reposilite/reposilite.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "reposilite",
+  "display": "Reposilite",
+  "type": "Other",
+  "install": [
+    {
+      "files": [
+        "https://maven.reposilite.com/releases/com/reposilite/reposilite/${version}/reposilite-${version}-all.jar"
+      ],
+      "type": "download"
+    }
+  ],
+  "run": {
+    "stop": "",
+    "command": "java -Xmx${memory}M -jar reposilite-${version}-all.jar --port ${port} --token ${token_name}:${secret}",
+    "workingDirectory": "",
+    "pre": [],
+    "post": [],
+    "environmentVars": {}
+  },
+  "data": {
+    "memory": {
+      "type": "integer",
+      "desc": "20MB: Minimal requirements for basic setup; 32MB: Repository for personal projects + CI + Proxy; 64MB: General use public repository; 128MB+: Huge repositories with high traffic & throughput",
+      "display": "Memory",
+      "required": true,
+      "value": "64",
+      "userEdit": true
+    },
+    "port": {
+      "type": "integer",
+      "desc": "Port for webgui",
+      "display": "Port",
+      "required": true,
+      "value": "8080",
+      "userEdit": true
+    },
+    "secret": {
+      "type": "string",
+      "desc": "Secret/Password for webgui",
+      "display": "Secret",
+      "required": true,
+      "value": "foo",
+      "userEdit": true
+    },
+    "token_name": {
+      "type": "string",
+      "desc": "Name of token",
+      "display": "Token Name",
+      "required": true,
+      "value": "foo",
+      "userEdit": true
+    },
+    "version": {
+      "type": "string",
+      "desc": "Version of Reposilite",
+      "display": "Version",
+      "required": true,
+      "value": "3.4.4",
+      "userEdit": true
+    }
+  },
+  "environment": {
+    "type": "standard"
+  },
+  "supportedEnvironments": [
+    {
+      "type": "standard"
+    }
+  ],
+  "requirements": {}
```

Comment:
Either specify java needs to be installed, or use javadl to download a valid Java version